packages/backend/src/services/UserService.ts

```diff
@@ -682,11 +682,20 @@ export class UserService {
                 );
             }
             // TODO: move to authorization service layer
+            // TODO we should probably remove the organization from the model
             const user = await this.userModel.getUserByPrimaryEmailAndPassword(
                 email,
                 password,
             );
-            this.identifyUser(user);
+            const userOrganization = this.loginToOrganization(
+                user.userUuid,
+                'password',
+            );
+            const userWithOrganization = {
```

Comment:
I believe we want to do this, add the organization to the user from the `loginToOrganization` instead of from the model.
If so, we should delete the code that already fetches the org for the user `userDetailsQueryBuilder`

Reply:
yes! good point! try removing this and see what the impact is